jnosql-lite/mapping-lite-processor/src/main/java/org/eclipse/jnosql/lite/mapping/repository/MethodMetadata.java

```diff
@@ -147,10 +147,10 @@ public String getEntityType() {
         return entityType;
     }
 
-    public Optional<Parameter> findPageable(){
+    public Optional<Parameter> findPageRequest(){
         for (Parameter parameter : this.parameters) {
             TypeElement element = parameter.type();
-            if("jakarta.data.page.Pageable".equals(element.getQualifiedName().toString())){
+            if(PageRequest.class.getName().equals(element.getQualifiedName().toString())){
```

Comment:
Wey better, thanks Max.

Reply:
I think when Class-File API comes - https://openjdk.org/jeps/457 - we can do something much better than mustache templates...I guess heheh